packages/monaco/src/browser/style/index.css

```diff
@@ -54,3 +54,8 @@
 .quick-open-entry .quick-open-row .monaco-icon-label .monaco-icon-label-description-container .monaco-highlighted-label .highlight {
     color: var(--theia-accent-color1);
 }
+
+.quick-open-entry .quick-open-row .monaco-icon-label .monaco-icon-label-description-container .label-description {
+    color: var(--theia-ui-font-color2);
+    font-size: calc(var(--theia-ui-font-size0) * 0.95) !important;
```

Comment:
If that !important bit really necessary? 
We should avoid it in the core as it forbids downstream users to override the style.

Reply:
sorry, it looks like it was leftover from testing